src/qos_enclave/Cargo.lock

```diff
@@ -728,6 +1321,36 @@ version = "1.9.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0fcc0b4a115bf80b728eb8ea024ad5bd707b615bfed49e0665b6e0f86fd082d9"
 
+[[package]]
+name = "httpdate"
+version = "1.0.3"
```

Comment:
This seems a little more questionable: https://github.com/pyfisch/httpdate

(but I had a detailed look at this small crate: all good)